pilot/pkg/model/push_context.go

```diff
@@ -717,18 +716,15 @@ func (ps *PushContext) getSidecarScope(proxy *Proxy, workloadLabels labels.Colle
 					}
 					return wrapper
```

Comment:
may be we can remove this return to avoid confusion?

Reply:
sure, will do